sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/CreateNamespaceSuite.scala

```diff
@@ -25,5 +25,7 @@ import org.apache.spark.sql.execution.command.v1
  */
 class CreateNamespaceSuite extends v1.CreateNamespaceSuiteBase with CommandSuiteBase {
   override def commandVersion: String = super[CreateNamespaceSuiteBase].commandVersion
-  override def alreadyExistErrorMessage: String = s"$notFoundMsgPrefix $namespace already exists"
+  override def alreadyExistErrorMessage: String =
+    s"$notFoundMsgPrefix $namespaceMessage already exists"
+  private def namespaceMessage: String = if (runningV1Command) s"$namespace" else s"'$namespace'"
```

Comment:
is there a way to unify it? e.g. we can try-catch the hive error and rethrow it with a new message.